### DISCUSSION TITLE
ERROR after cloning git

### DISCUSSION THREAD

**Discussion by sheebz1** [jschuh/klipper-macros#144] [General]
![image](https://github.com/jschuh/klipper-macros/assets/87398332/2e6b7be2-d34f-4a79-a16c-3af0b061290c)

not sure how to get around this. i cant find anything involving m300 in any of my config files aside from the klipper-macros beep.cgf.

any insight on this is greatly appreciated!

**Comment by jschuh** [comment]
Some of the Klipper forks define an M300 command. It's either that or there's an M300 defined by something else in your config.

You're not the first person to hit this, which is why I've been thinking about making M300 optional ([the branch is here](https://github.com/jschuh/klipper-macros/tree/beeps)).

**Reply by sheebz1** [reply]
Ok, so I uploaded the new branch. It still came up with the same M300 error, so i just commented out the beep config. Restarted klipper, and now this is where I got stuck before when I deleted the entire beep config all together 
![image](https://github.com/jschuh/klipper-macros/assets/87398332/6452e4af-11af-43db-ac7e-272c6a768784)

![image](https://github.com/jschuh/klipper-macros/assets/87398332/27f5e855-aa25-4540-84bf-8f959708467d)

All I can find online regarding this error is to comment out the "#rename_existing: SET_PRINT_STATS_INFO_BASE" line from the mailsail config.
see here - [https://www.reddit.com/r/klippers/comments/11hri1m/help_please_this_is_a_new_one_for_me_existing/](url)

Mine was already commented out, so I tried uncommenting the line. No change. commenting it back out, saving and rebooting system. No change. Commenting out the entire section, still no change.

Do you have any idea where I should go from here?

**Reply by jschuh** [reply]
That means the fork you're using is too old to support the SET_PRINT_STATS_INFO command (which Klipper added nearly a year ago). The macros require this command, so I'd suggest either switching to mainline Klipper or seeing if there's a newer version of the fork you're using.

**Comment by sheebz1** [comment]
Oh perfect!

Ya I can’t find anything else related to m300 in my config, so not sure. I’m a klipper noob though so could be over looking something.

I tried deleting the beep config all together, but that started a whole new line of errors so I just removed it for now.

I will try the new branch you sent later on

Thanks!!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
